src/System.IO.Pipelines/tests/StreamPipeReaderTests.cs

```diff
@@ -529,6 +529,17 @@ public void InvalidMinimumReadSizeThrows()
             Assert.Throws<ArgumentOutOfRangeException>(() => new StreamPipeReaderOptions(minimumReadSize: 0));
         }
 
+        [Fact]
+        public void LeaveUnderlyingStreamOpen()
+        {
+            var stream = new MemoryStream();
+            var reader = PipeReader.Create(stream, new StreamPipeReaderOptions(leaveOpen: true));
```

Comment:
Style: Factory methods, like `PipeReader.Create` are not considered acceptable for `var` in corefx.

```suggestion
            PipeReader reader = PipeReader.Create(stream, new StreamPipeReaderOptions(leaveOpen: true));
```